### ISSUE TITLE
Q scores affected by subsampling of .fastq?

### ISSUE THREAD

**Issue by GeoMicroSoares** [wdecoster/NanoPlot#14]
Hi @wdecoster, here's the follow up from our brief talk on the Issues section of pauvre.

So, after filtering my .fastq to hold only reads with a Phred score >9 and plotting them using the default Nanoplot settings I get:
![bp_v1_qf9_nanoplot](https://user-images.githubusercontent.com/14272488/31662369-030906e8-b336-11e7-9d94-01a4283effbc.png)
The weird view is due to a bigger 7k read as you can see below in the Nanostats.txt output from that command:
```
$ cat nanoplot_BP_v1_qf9/NanoStats.txt
Number of reads:        210
Total bases:    93602
Median read length:     305.5
Mean read length:       445.72
Readlength N50: 666

Top 5 read lengths and their average basecall quality score:
Length: 7020bp  Q: 11.66
Length: 2172bp  Q: 11.6
Length: 1507bp  Q: 11.56
Length: 1366bp  Q: 11.42
Length: 1299bp  Q: 11.51
```

Now in order to get a cleaner view of this dataset, I tweaked the NanoPlot command to --maxlength 6000. This results in the following ['pauvre-like' plot](https://github.com/conchoecia/pauvre/issues/21).
![figure_1](https://user-images.githubusercontent.com/14272488/31662488-71a009e4-b336-11e7-9100-cacb0a10dc40.png)

Interestingly, in this command's Nanostats output here's what I get:
```
3$ cat nanoplot_BP_v1_qf9_filt6kb/nanoplot_BP_v1_qf9_filt6kb_NanoStats.txt
Number of reads:        210
Total bases:    93602
Median read length:     305.5
Mean read length:       445.72
Readlength N50: 541

Top 5 read lengths and their average basecall quality score:
Length: 7020bp  Q: 7.26
Length: 2172bp  Q: 7.03
Length: 1507bp  Q: 7.21
Length: 1366bp  Q: 7.15
Length: 1299bp  Q: 7.06
```

What am I missing here? I thought the Q was read-specific and its calculation wasn't dependent on the remainder reads as it seems it's happening here.

I'll open another issue regarding the x-axis changes with the log view ASAP.

Thanks in advance!!

**Comment by wdecoster** [comment]
I have checked quickly and can't reproduce this issue. Most puzzling is that any filtering (such as --maxlength 6000) is done only after calculating stats. I understand that you cannot share your fastq file, but could you add the exact commands you used?

**Comment by wdecoster** [comment]
Feel free to reopen this in case you can replicate the issue. Thanks!
